core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStoreDir.java

```diff
@@ -146,4 +147,14 @@ private Optional<PageId> getPageId(Path path) {
       return Optional.empty();
     }
   }
+
+  @Override
```

Comment:
We can define the default function body for this, so we don't have override it with duplicated empty function.

Reply:
Reduced boilerplate a bit in ea969793